### PR TITLE
Describe team performance measurement process

### PR DESCRIPTION
**Pull request by rumyantseva**
# Description

This is a proposal for our process improvement. Feel free to correct and/or rephrase it.

## Readiness checklist

* [ ] I added tests for new functionality or bugfixes.
* [ ] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)),
      Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: .github/PROCESS.md [hunk 1]
```diff
@@ -26,3 +26,19 @@ This guide tries to be short and does not mention things that are completely aut
...
    Autogenerated GitHub's squash commit's body should be manually replaced by "Refs #{issue_number}." or "Closes #{issue_number}.".
    Do not leave original commit messages in the squashed commit body.
+
+## Team Performance Metrics
+
+1. We want to deliver fast and make our deliverables predictable.
+2. To achieve that, we measure the flow of work and estimate tasks.
+3. The flow of work is measured automatically based on GitHub's pull request workflow.
+4. Tasks are measured manually by engineers during Sprint planning.
+5. Task estimation depends on the scope, the complexity of the task, and the possible risks and clarity.
```
Comment (rumyantseva) [inline .github/PROCESS.md:36]:
```suggestion
5. Task estimation depends on the following parameters:
   * Scope
   * Complexity of the task
   * Possible risks
   * Clarity
```

@AlekSi @noisersup please check if this description for 5 looks better.

Comment (noisersup) [reply .github/PROCESS.md:36]: LGTM!

Comment (AlekSi) [reply .github/PROCESS.md:36]: I don't think that risks worth a separate point. We always have risks. We can have extra risks due to complexity on unclarity, but other points cover that already

Comment (AlekSi) [reply .github/PROCESS.md:36]: I will update the PR

**Review by w84thesun — APPROVED** [review]
LGTM!

**Review by AlekSi — DISMISSED** [review]

**Review by w84thesun — APPROVED** [review]

**Review by noisersup — APPROVED** [review]

**Review by AlekSi — APPROVED** [review]

**Pull request merged**